webapp/channels/src/components/tabs/tab/index.tsx

```diff
@@ -10,15 +10,17 @@ type Props = {
     title?: React.ReactNode | undefined;
     unmountOnExit?: boolean | undefined;
     tabClassName?: string | undefined;
+    tabIndex?: number;
 }
 
-export default function Tab({children, title, unmountOnExit, tabClassName, eventKey}: Props) {
+export default function Tab({children, title, unmountOnExit, tabClassName, eventKey, tabIndex = -1}: Props) {
```

Comment:
I'm not sure if this is right since it makes the tabs unfocusable by default unless you pass a `tabIndex`

Reply:
Oh, Community hasn't been updated to include these changes. Maybe this works now, but I haven't had a chance to try this out yet

Reply:
ok, @hmhealey thanks for the feedback. We can go back to implement your suggestion in a following ticket.